### DISCUSSION TITLE
Sax Feaures incompatibel with fasterxml woodtstox

### DISCUSSION THREAD

**Discussion by pwachira** [aws/aws-sdk-java#2829] [General]
Fasterxml woodstox does not support the sax feature `http://apache.org/xml/features/disallow-doctype-decl`
and results in an exception if woodtox is the sax driver found in classpath.
To fix this I had to add the jvm option` -Dorg.xml.sax.driver=org.apache.xerces.parsers.SaxParser`

Is there a way to use this as the default - maybe in META-INF?

**Comment by debora-ito** [comment]
I don't think there's a way to set another Sax Parser as default in the SDK. Glad you found the workaround then.

This is a duplicate of https://github.com/aws/aws-sdk-java/issues/2830.
